src/Microsoft.Diagnostics.ExtensionCommands/DumpAsyncCommand.cs

```diff
@@ -3,6 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 #nullable enable
+#pragma warning disable CS8600
```

Comment:
If we are going to use nullable enable, we shouldn't block these. It's contradictory immediately.

Reply:
I will fix these errors the right way but I just wanted to get past them for now.  FYI, nullable was already enabled, but the 8.0 SDK caused a bunch of warnings (as errors) that weren't being flagged before.